lib/ace/ext/papeeria/spellchecker_popup.coffee

```diff
@@ -16,7 +17,8 @@ define([
   # @param {Array} corrections: array of strings with substitution options.
   # @return {Array}: array of JSONs, actually.
   convertCorrectionList = (corrections) ->
-    return ({caption: item, value: item, meta: item, meta_score: corrections.length - i} for item, i in corrections)
+    # TODO: get server to provide corrections' source
+    return ({caption: item, value: item, meta: "spellcheck", score: corrections.length - i} for item, i in corrections)
```

Comment:
replace "spellcheck" with empty or whitespace-only string, otherwise the user gonna see a column of "spellcheck" in his popup